install/mysql/glpi-empty.sql

```diff
@@ -151,7 +151,7 @@ CREATE TABLE `glpi_apiclients` (
   `app_token` VARCHAR(255) DEFAULT NULL,
   `app_token_date` timestamp NULL DEFAULT NULL,
   `dolog_method` TINYINT NOT NULL DEFAULT '0',
-  `comment` TEXT,
+  `comment` text COLLATE utf8_unicode_ci,
```

Comment:
This one should be added in migration aswell, no? I do not knwon what was used when nothing was define; (but I won't bet on the one we'd like :D

Reply:
(same for `glpi_appliance*`)